EvidenceApi/V1/Boundary/Request/AuditEventRequest.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace EvidenceApi.V1.Boundary.Request
+{
+    public class AuditEventRequest
```

Comment:
Usually we use `Boundary/Request` for classes which controller's consume from the HTTP request so not sure this class should be here